src/scripts/settings/Settings.ts

```diff
@@ -89,6 +89,14 @@ Settings.add(new MultipleChoiceSetting('breedingDisplay', 'Breeding progress dis
     ],
     'percentage'
 ));
+Settings.add(new MultipleChoiceSetting('shopButtons', 'Shop amount buttons:',
+    [
+        new SettingOption('+10, +100', 'original'),
+        new SettingOption('+100, +1000', 'bigplus'),
+        new SettingOption('\u00d710, \u00f710', 'multiplication'),
```

Comment:
I'm not sure the divider option makes sense here, would a ×100 option not be more useful?
```suggestion
        new SettingOption('×10, ×100', 'multiplication'),
```
If not, I think we should just use the actual icons here, so we can tell what they are at a glance.
```suggestion
        new SettingOption('×10, ÷10', 'multiplication'),
```